packages/editor/src/components/rich-text/index.js

```diff
@@ -914,6 +883,7 @@ export class RichText extends Component {
 						</Fragment>
 					) }
 				</Autocomplete>
+				<RemoveBrowserShortcuts />
```

Comment:
Another instance of: Should every `RichText` be creating these global event bindings? Worse yet, wouldn't this also disable handling of those shortcuts when not even within a `RichText`, e.g. the title input?

Reply:
You're right. Maybe the place to put these would be more up the tree in the Editor component to disable it for all fields. No reason to bind multiple times. In the case af the title, disabling seems wanted as well, otherwise undoing with the browser will add levels in Gutenberg.